### ISSUE TITLE
cqlsh truncating timestamps to Second resolution? (Plus conversion function toUnixTimestamp not found)

### ISSUE THREAD

**Issue by dybal** [scylladb/scylla#2915]
*Installation details*
Scylla version (or git commit hash): 2.0.0 (docker version)
Cluster size: 1
OS (RHEL/CentOS/Ubuntu/AWS AMI): docker Version

Platform (physical/VM/cloud instance type/docker): docker
Resources= docker defaults

Created a table:

```
CREATE TABLE test (
	pk int,
	instant timestamp,
	PRIMARY KEY(pk)
);
```

Inserted some rows:

```
INSERT INTO test(pk, instant) values (1, '2017-10-01 03:00:00+0000');
INSERT INTO test(pk, instant) values (2, '2017-10-01 03:00:00.123+0000');
```

Selected:

```
cqlsh:simet> select * from test;

 pk | instant
----+--------------------------
  1 | 2017-10-01 03:00:00+0000
  2 | 2017-10-01 03:00:00+0000
```

Tried the toUnixTimestamp function to get the timestamp as a number to check if it's cqlsh that truncates the timestamp, but got an error:

```
cqlsh:simet> select pk, toUnixTimestamp(instant) from test;
InvalidRequest: Error from server: code=2200 [Invalid query] message="Unknown function 'tounixtimestamp'" 
```

Tried all of this in a Cassandra:latest docker, cqlsh doesn't truncate:

```
cqlsh:simet> select * from test;
 pk | instant
----+---------------------------------
  1 | 2017-10-01 03:00:00.000000+0000
  2 | 2017-10-01 03:00:00.123000+0000

cqlsh:simet> select pk, tounixtimestamp(instant) from test;

 pk | system.tounixtimestamp(instant)
----+---------------------------------
  1 |                   1506826800000
  2 |                   1506826800123

```
Tried all of this in a Cassandra:2.2 docker, cqlsh truncates, but the toUnixTimestamp is found:

```
cqlsh:simet> select * from test;

 pk | instant
----+--------------------------
  1 | 2017-10-01 03:00:00+0000
  2 | 2017-10-01 03:00:00+0000

(2 rows)

cqlsh:simet> select pk, toUnixTimestamp(instant) from test;

 pk | system.tounixtimestamp(instant)
----+---------------------------------
  1 |                   1506826800000
  2 |                   1506826800123
```

**Comment by tzach** [comment]
@penberg can you please take a look?

**Comment by slivne** [comment]
Scylla's cqlsh does not support this at the moment - we have an issue to add timestamp/timeuuid functions
#2950 - we will add all of them

**Comment by tzach** [comment]
added an issue to track toUnixTimestamp #3567

**Comment by tzach** [comment]
@slivne what is left to do here?
toUnixTimestamp #3567 is fixed in 2.3
CQLSh issue was fixed in move to Cassandra 3.11 Java Tools #3686

**Comment by slivne** [comment]
closing the issue should be fixed
